src/node/blockstorage.cpp

```diff
@@ -1091,4 +1187,18 @@ void ImportBlocks(ChainstateManager& chainman, std::vector<fs::path> vImportFile
         }
     } // End scope of ImportingNow
 }
+
+std::ostream& operator<<(std::ostream& os, const BlockfileType& type) {
+    switch(type) {
+        case BlockfileType::NORMAL: os << "normal"; break;
+        case BlockfileType::ASSUMED: os << "assumed"; break;
+        default: os.setstate(std::ios_base::failbit);
```

Comment:
This would be resolved by @ryanofsky 's effort to remove the chainstate names/types entirely, as discussed here and linked at the bottom: https://github.com/bitcoin/bitcoin/pull/28562#discussion_r1344824078